www_src/pages/editor/link-editor.jsx

```diff
@@ -0,0 +1,62 @@
+var React = require('react/addons');
+var Binding = require('../../lib/binding.jsx');
+var ColorGroup = require('../../components/color-group/color-group.jsx');
+var Range = require('../../components/range/range.jsx');
+
+var LinkEditor = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin,
+    Binding
+  ],
+  getInitialState: function () {
+    return {
+      borderRadius: 5,
+      color: "#69A0FC",
+      textColor: "white",
```

Comment:
I think what we actually need here is to compute whether white or black has greater contrast from the background color -- there are a bunch of libraries that can handle this such as https://www.npmjs.com/package/color. It should be pretty simple to do. would you mind adding that in?